test_unstructured/partition/test_auto.py

```diff
@@ -324,7 +325,7 @@ def test_auto_partition_pdf_from_filename(pass_metadata_filename, content_type,
     )
 
     idx = 3
-    assert isinstance(elements[idx], Title)
+    assert isinstance(elements[idx], Text)
```

Comment:
@qued - Any thoughts on a setting that could have caused this to swap in the docker container?